hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdateProcessorSupplier.java

```diff
@@ -94,6 +100,11 @@ public void init(@Nonnull Context context) {
                 .thenApply(m -> Traversers.empty());
     }
 
+    @Override
+    public Permission permission() {
+        return new MapPermission(mapName, ACTION_CREATE, ACTION_PUT, ACTION_REMOVE);
```

Comment:
Can it really remove? https://github.com/hazelcast/hazelcast/blob/b1a5fd7fdaa6d29b76d08577cc57a7e43e034b68/hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdatingEntryProcessor.java#L76

Reply:
From what I know UPDATE=REMOVE + PUT @gurbuzali

Reply:
We can't update the key, so it's just PUT i guess. And we specifically don't allow updating the value to `null`, which would be REMOVE. So I think it's just PUT.